crossdock/client/errors/behavior.go

```diff
@@ -311,11 +312,29 @@ func Run(t crossdock.T) {
 			wantBodyStartsWith: `UnexpectedError: error for procedure "phone" of service "yarpc-test": ` +
 				`UnexpectedError: error for procedure "unexpected-error" of service "yarpc-test": error` + "\n",
 		},
+		{
+			name: "remote timeout",
+			headers: map[string]string{
+				"RPC-Caller":     "yarpc-test",
+				"RPC-Service":    "yarpc-test",
+				"RPC-Procedure":  "waitfortimeout/raw",
+				"RPC-Encoding":   "raw",
+				"Context-TTL-MS": "100",
+			},
+			wantStatus: 504,
+			skipStatus: 400,
+			wantBodyStartsWith: `handler timeout for procedure "waitfortimeout/raw"` +
+				` of service "yarpc-test" from caller "yarpc-test" after`,
+		},
 	}
 
 	for _, tt := range tests {
 		res := client.Call(t, tt.headers, tt.body)
 		t.Tag("scenario", tt.name)
+		if res.Status == tt.skipStatus {
+			t.Skipf("waitfortimeout/raw method not implemented: %v", res.Body)
```

Comment:
Don't hard code the method name ("waitfortimeout/raw"). Just use a generic message with `tt.name`. That will give us enough information about what got skipped and why.